### PR TITLE
Add inTransaction() to ConnectionInterface

### PR DESCRIPTION
**Pull request by Koopzington**
PR for #10

### CODE REVIEW

**Comment by grizzm0** [comment]
You should never create a PR from your own master. Create a new branch per fix/PR and send that as a PR.

**Comment by weierophinney** [comment]
@Koopzington Please address the CS concerns, which you can see starting here:

- https://travis-ci.org/zendframework/zend-db/jobs/68619938#L359

File: src/Adapter/Driver/ConnectionInterface.php [hunk 1]
```diff
@@ -68,6 +68,13 @@ public function commit();
     public function rollback();
 
     /**
+     * Checks whether the connection is in transaction state.
+     *
+     * @return boolean
+     */
+    public function inTransaction();
+
```
Comment (grizzm0) [inline src/Adapter/Driver/ConnectionInterface.php]: This is where the error is. You've got 4 spaces here which should be removed.

**Comment by Koopzington** [comment]
closing PR due to #20

**Pull request closed without merging**